### PR TITLE
Update Report Usage Doc to Mention Bearer Token also

### PR DESCRIPTION
**Pull request by cookieMr**
This commit is a result of a discussion in #375

### CODE REVIEW

File: packages/web/docs/pages/specs/usage-reports.md [hunk 1]
```diff
@@ -9,8 +9,12 @@ The official JavaScript Hive Client (`@graphql-hive/client`) collects executed o
 | -------- | ------------------------------------ |
 | Endpoint | `https://app.graphql-hive.com/usage` |
 | Header   | `X-API-Token: token-here`            |
+| Header   | `Authorization: Bearer token-here`   |
```
Comment (kamilkisiela) [inline packages/web/docs/pages/specs/usage-reports.md:12]:
```suggestion
| Header   | `Authorization: Bearer token-here`   |
```

File: packages/web/docs/pages/specs/usage-reports.md [hunk 2]
```diff
@@ -9,8 +9,12 @@ The official JavaScript Hive Client (`@graphql-hive/client`) collects executed o
...
 | Method   | `POST`                               |
 
+> The same token can be used in either `Authorization` or `x-api-token`
+> but use only once authentication header.
+
```
Comment (kamilkisiela) [inline packages/web/docs/pages/specs/usage-reports.md:17]:
```suggestion
```

File: packages/web/docs/pages/specs/usage-reports.md [hunk 3]
```diff
@@ -286,3 +290,6 @@ curl -X POST \
   -H 'content-type: application/json' \
   -d '{ "size": 1, "map": { "aaa": { "operationName": "me", "operation": "query me { me { id } }", "fields": ["Query", "Query.me", "User", "User.id"] } }, "operations": [{ "operationMapKey" : "c3b6d9b0", "timestamp" : 1663158676535, "execution" : { "ok" : true, "duration" : 150000000, "errorsTotal" : 0 }, "metadata" : { "client" : { "name" : "demo" , "version" : "0.0.1" } } } ] }'
 ```
+
+`x-api-token` header can be replaced with `Authorization` and bearer token
+(the same token as used with `x-api-token`).
```
Comment (kamilkisiela) [inline packages/web/docs/pages/specs/usage-reports.md:295]:
```suggestion
```

**Review by kamilkisiela — CHANGES_REQUESTED** [review]
The `x-api-token` header is deprecated. Maybe it's better to remove it from the spec?

**Comment by cookieMr** [comment]
I can quickly update this doc.

File: packages/web/docs/pages/specs/usage-reports.md (hunk 2 above)
Comment (cookieMr) [reply packages/web/docs/pages/specs/usage-reports.md]: removed

**Review by kamilkisiela — APPROVED** [review]

**Pull request merged**